bokeh/io/output.py

```diff
@@ -91,7 +91,7 @@ def output_file(filename: PathLike, title: str = "Bokeh Plot",
 def output_notebook(resources: Resources | None = None, verbose: bool = False,
         hide_banner: bool = False, load_timeout: int = 5000, notebook_type: NotebookType = "jupyter") -> None:
     ''' Configure the default output state to generate output in notebook cells
-    when :func:`show` is called. Note that, :func:`show` may be called multiple
+    when :func:`show` is called. Note that :func:`show` may be called multiple
```

Comment:
could these be `|show|` and `|save|` now?

Reply:
totally!